### PR TITLE
Improve formatting ux

### PR DESCRIPTION
**Pull request by Erfanm83**
Hi @dnnrly,

As part of Hacktoberfest, I've made the following improvements based on the discussion in issue #40 (Additional formatting):

- Added borders to the description section to enhance readability.
- Changed the status and summary section to a status bar using the `lipgloss` package.
- Fixed failing tests caused by the UX changes.

Please let me know if there's any additional feedback or suggestions.

#40 
Thanks,
@Erfanm83

### CODE REVIEW

File: httpref_test.go [hunk 1]
```diff
@@ -62,19 +62,23 @@ func TestReferences_InRange(t *testing.T) {
...
+	// Update the expected value to match the new styled format
+	expected := lipgloss.JoinVertical(
+		lipgloss.Bottom,
+		nameStyle.Render(reference.Name),
+		summaryStyle.Render(reference.Summary),
+		renderStatusBar(reference.Name, reference.Summary),
+	)
+
+	if actual != expected {
+		t.Errorf("Expected %v, but got %v", expected, actual)
```
Comment (dnnrly) [inline httpref_test.go:80]: Can we use `assert.Equal` here?

File: view.go [hunk 2]
```diff
@@ -45,21 +49,20 @@ func renderStyles() {
...
 	glamour.LightStyleConfig.Document.Margin = &margin
 	glamour.DarkStyleConfig.Text.Color = &descriptionForeColorDarkTheme
 	glamour.LightStyleConfig.Text.Color = &descriptionForeColorLightTheme
-
-	// styled using adaptive color to set one of the two colors based on the current theme
-	summaryStyle = resultStyle.Copy().
-		Foreground(lipgloss.AdaptiveColor{Light: "20", Dark: "178"})
+	// if background color intended
+	// backgroundColor := "#353533"
+	// glamour.DarkStyleConfig.Document.BackgroundColor = &backgroundColor
```
Comment (dnnrly) [inline view.go:65]: Please can this commented out code be removed?

**Review by dnnrly — APPROVED** [review]

**Comment by dnnrly** [comment]
This is a great addition to `httpref`! Only some small amendments requested but otherwise a solid contribution. I might raise an issue with some additional work - but feel free to boost your Hacktoberfest PRs if you're up for it.

**Review by dnnrly — CHANGES_REQUESTED** [review]
Small changes required.

**Review by Erfanm83 — COMMENTED** [review]
Hi I have updated the code based on your reviews
Thanks for reveiwing if you had more of that let me know

**Review by dnnrly — APPROVED** [review]

**Comment by dnnrly** [comment]
Thank you for your contribution! As is traditional for me, please let me know you Twitter username (if you have one) for me to start following you.

**Comment by Erfanm83** [comment]
Thanks @dnnrly for meging my updates!
I will be so glad if you want more changes to this project 
It looks like there is an issue of panic usage throughout the code, I want to fix that bug anyway

**Comment by Erfanm83** [comment]
> Thank you for your contribution! As is traditional for me, please let me know you Twitter username (if you have one) for me to start following you.

No I hadn't Twitter(X) Account . But I Created one with this username : @Erfanm_2004
I Will add my Twitter Profile on my Github Account soon .

**Pull request merged**